pages/api/webhook.ts

```diff
@@ -0,0 +1,61 @@
+import Stripe from "stripe";
+import { buffer } from "micro";
+import { NextApiRequest, NextApiResponse } from "next";
+import { ThirdwebSDK } from "@thirdweb-dev/sdk";
+import { EDITION_ADDRESS } from "../../constants/addresses";
+
+const stripe = new Stripe(process.env.STRIPE_SECRET_KEY as string, {
+  apiVersion: "2022-11-15",
+});
+
+const webhookSecret = process.env.WEBHOOK_SECRET_KEY as string;
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
+  const buf = await buffer(req);
+  const sig = req.headers["stripe-signature"];
```

Comment:
Haven't read up on this webhook process, but assuming stripe signs their message before sending it to our webhook endpoint and that's the validation you're doing below to secure this endpoint. I'd mention the webhook setup/how it works at a high level (not the signature piece but just the fact that we setup stripe to ping our endpoint) in the README.

Reply:
oh okay. I mentioned all this in the guide haha but will add in readme as well!